moler/device/textualdevice.py

```diff
@@ -251,21 +254,29 @@ def from_named_connection(cls, connection_name):
     def __del__(self):
         self._stop_prompts_observers()
 
-    def _collect_cmds_for_state_machine(self):
-        for state in self._get_available_states():
-            self._cmdnames_available_in_state[state] = dict()
+    def _load_cmds_for_state(self, state):
+        self._cmdnames_available_in_state[state] = dict()
+        cmds = self._collect_cmds_for_state(state)
+        self._cmdnames_available_in_state[state].update(cmds)
 
-            cmds = self._collect_cmds_for_state(state)
+    def _load_events_for_state(self, state):
```

Comment:
same as above - change name to `_load_eventnames_for_state()`

Reply:
Done.